test/Parse/ConditionalCompilation/compiler_version.swift

```diff
@@ -7,7 +7,7 @@
   asdf asdf asdf asdf
 #endif
 
-#if _compiler_version("10.*.10.10")
+#if _compiler_version("600.*.10.10")
```

Comment:
Grepped for low version numbers myself—it looks like there's also 3 instances of `_compiler_version("4.*.0")` in `test/Parse/ConditionalCompilation/identifierName.swift` that you might want to update.

Reply:
`_compiler_version("4.*.0")` will still actually work correctly; it's just turned into a really silly example, since it's basically testing for compiler version `0.004`.I'm going to leave it that way in the interest of expedience.